packages/bpk-component-button/stories.js

```diff
@@ -46,364 +46,80 @@ const cssModules = (styles = {}) => className =>
 
 const getClassName = cssModules(STYLES);
 
+const ButtonStory = ({ className, ...rest }: { className: ?string }) => (
```

Comment:
I rewrote most of this file as it had a lot of duplication from way back - it's neater now and removes the need for a separate 'iconOnly' story. It's also now possible to see all buttons on screen together:

![Screen Shot 2019-07-22 at 08 17 59](https://user-images.githubusercontent.com/73652/61613741-0b695f80-ac5a-11e9-89b3-6aab6191fb3f.png)